libs/lib-client/installer/src/lib/installer.service.ts

```diff
@@ -0,0 +1,11 @@
+import { EventEmitter, Injectable } from '@angular/core';
+import { ActivitySchema } from '@flogo-web/core';
+
+@Injectable()
+export class InstallerService {
+  contribInstalled = new EventEmitter<ActivitySchema>();
```

Comment:
We should probably use `ContribSchema` instead of `ActivitySchema` since the installation could have been for a trigger or function.

Reply:
Done.